cmd/cluster/health.go

```diff
@@ -177,15 +205,27 @@ func (o *healthOptions) run() error {
 	if err != nil {
 		log.Fatalf("error: %v", err)
 	}
+	fmt.Fprintf(o.IOStreams.Out, "\n \n")
 	fmt.Printf(string(d))
```

Comment:
I know this wasn't changed here, but can we use a more descriptive name for the health output string than `d`?  Maybe something like `healthOutput`?

Reply:
Done.